db/schema.rb

```diff
@@ -11,10 +11,10 @@
 #
 # It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20210308053346) do
+ActiveRecord::Schema.define(:version => 20210317030546) do
 
   create_table "students", :force => true do |t|
```

Comment:
Is `:force => true` necessary?

Reply:
Probably not.. I didn't indicate it in my migration file so I'm not sure why it didn't default to false. According to the [docs](https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-create_table) it drops the table before creating it. Ensures a clean db I guess?

Reply:
Yeah if its just something rails did when generating the schema that's weird. There isn't much documentation on it apart from saying it defaults to false. I just checked the `schema.rb` from my project and it has the same thing. I guess it actually defaults to true in the schema? 😅

Reply:
yeah it must..  🤔